Lib/concurrent/futures/process.py

```diff
@@ -49,7 +49,6 @@
 from concurrent.futures import _base
 import queue
 import multiprocessing as mp
-import multiprocessing.connection
```

Comment:
This one is actually necessary. It is later relied on when `mp.connection.wait` is called on L407. Without this import we are not guaranteed that `import multiprocessing as mp` causes `mp.connection` to exist.

Reply:
Good catch. It concerns me that no tests appeared to fail when this import was removed.

Reply:
Oh, I suppose `multiprocessing.connection` was still imported through some side effect somewhere...

Reply:
Yeah, and when that code gets changed this might break. Maybe add a comment? (This is the kind of thing that pyright is very picky about, and in this case I agree -- we were bitten by this kind of bug at Dropbox numerous times.)

Reply:
Thoughts on b3488e8?

Reply:
A comment might be more helpful than using an odd idiom to please a tool.

Reply:
For the record, I pushed the change because I personally find it easier to understand the code if the import is more explicit, not to please a tool :)

But happy to revert the commit, and add a comment instead!

Reply:
Added a comment in 4269138